docs/features/pytorch.md

```diff
@@ -0,0 +1,140 @@
+# Pytorch Integration Guide
+
+## Overview
+This guide explains how to integrate PyTorch with `pixi`, it supports multiple ways of installing PyTorch.
+
+- Install PyTorch using `conda-forge` Conda channel (Recommended)
+- Install PyTorch using `pypi`, using our `uv`'s integration. (Most versions available)
+- Install PyTorch using `pytorch` Conda channel (Legacy)
+
+With these options you can choose the best way to install PyTorch based on your requirements.
+
+## Installing from Conda-forge
+You can install PyTorch using the `conda-forge` channel.
+These are the community maintained builds of PyTorch.
+You can make direct use of the Nvidia provided packages to make sure the packages can work together.
+
+!!! note
+    Currently not well-supported for Windows, but there is lots of work being done to get this working.
+    Follow the work on the [feedstock](https://github.com/conda-forge/pytorch-cpu-feedstock)
+!!! note
+    Pixi uses the `system-requirements.cuda` to tell it can use the `cuda` packages.
+    Without it, pixi will install the `cpu` versions.
+
+=== "`pixi.toml`"
+    ```toml title="Bare minimum conda-forge pytorch with cuda installation"
+    --8<-- "docs/source_files/pixi_tomls/pytorch-conda-forge.toml:minimal"
+    ```
+=== "`pyproject.toml`"
+    ```toml title="Bare minimum conda-forge pytorch with cuda installation"
+    --8<-- "docs/source_files/pyproject_tomls/pytorch-conda-forge.toml:minimal"
+    ```
+
+To deliberately install a specific version of the `cuda` packages you can depend on the `cuda-version` package which will then be interpreted by the other packages at resolve time.
```

Comment:
Maybe add a closed note, with an explanation why you need `cuda-version` and the system-requirements?

Reply:
Added some more information, hopefully that helps, I added a `note` for the system requirements

Reply:
Hmm still not sure if I get it. So if I put the `system-requirements` to a lower cuda, why do I still get a higher cuda pytorch variant for example? I would just expect that to work.

Reply:
So maybe it will be more accurate to say, that some packages like `PyTorch` (instead of cudatoolkit) do not use the cuda system-requirement but instead put a constraint on the cuda-version? And that is is somewhat unspecified what will happen if you specify only `__cuda`?

Reply:
System requirements are not real version specifications. You're describing the system pixi can solve for. 
None of the packages in this case actually use the cuda version and often only depend on `__cuda = "*"` Even though they are using a higher version than you specify. 

But it really shows what a bad design this is. We need to fix this ASAP. 

I'll take a look if I can make it more clear.

Reply:
I do see a requirement for cudatoolkit though?